src/scripts.js

```diff
@@ -17,40 +17,47 @@ import Activity from './Activity';
 var historicalWeek = document.querySelectorAll('.historicalWeek');
-var sleepToday = document.getElementById('sleepToday');
-var sleepQualityToday = document.getElementById('sleepQualityToday');
-var avUserSleepQuality = document.getElementById('avUserSleepQuality');
-var sleepThisWeek = document.getElementById('sleepThisWeek');
-var sleepEarlierWeek = document.getElementById('sleepEarlierWeek');
-var friendChallengeListToday = document.getElementById('friendChallengeListToday');
-var friendChallengeListHistory = document.getElementById('friendChallengeListHistory');
-var bigWinner = document.getElementById('bigWinner');
-var userStepsToday = document.getElementById('userStepsToday');
-var avgStepsToday = document.getElementById('avgStepsToday');
-var userStairsToday = document.getElementById('userStairsToday');
-var avgStairsToday = document.getElementById('avgStairsToday');
-var userMinutesToday = document.getElementById('userMinutesToday');
-var avgMinutesToday = document.getElementById('avgMinutesToday');
-var userStepsThisWeek = document.getElementById('userStepsThisWeek');
-var userStairsThisWeek = document.getElementById('userStairsThisWeek');
-var userMinutesThisWeek = document.getElementById('userMinutesThisWeek');
-var bestUserSteps = document.getElementById('bestUserSteps');
-var streakList = document.getElementById('streakList');
-var streakListMinutes = document.getElementById('streakListMinutes')
+// var sleepToday = document.getElementById('sleepToday');
+// var sleepQualityToday = document.getElementById('sleepQualityToday');
+// var avUserSleepQuality = document.getElementById('avUserSleepQuality');
+// var sleepThisWeek = document.getElementById('sleepThisWeek');
+// var sleepEarlierWeek = document.getElementById('sleepEarlierWeek');
+// var friendChallengeListToday = document.getElementById('friendChallengeListToday');
+// var friendChallengeListHistory = document.getElementById('friendChallengeListHistory');
+// var bigWinner = document.getElementById('bigWinner');
+// var userStepsToday = document.getElementById('userStepsToday');
+// var avgStepsToday = document.getElementById('avgStepsToday');
+// var userStairsToday = document.getElementById('userStairsToday');
+// var avgStairsToday = document.getElementById('avgStairsToday');
+// var userMinutesToday = document.getElementById('userMinutesToday');
+// var avgMinutesToday = document.getElementById('avgMinutesToday');
+// var userStepsThisWeek = document.getElementById('userStepsThisWeek');
+// var userStairsThisWeek = document.getElementById('userStairsThisWeek');
+// var userMinutesThisWeek = document.getElementById('userMinutesThisWeek');
+// var bestUserSteps = document.getElementById('bestUserSteps');
+// var streakList = document.getElementById('streakList');
+// var streakListMinutes = document.getElementById('streakListMinutes')
+// let hydroChart = document.getElementById("hydroChart")
+// let sleepChart = document.getElementById("sleepChart")
+
+
```

Comment:
Awesome job of moving of all these global DOM variables into their functions! Looks super clean!